wire-ios-sync-engine/Tests/Source/UserSession/ZMUserSessionTests.swift

```diff
@@ -94,12 +105,17 @@ class ZMUserSessionTests: ZMUserSessionTestsBase {
             self.createSelfClient()
         }
 
-        // when
-        sut.didRegisterSelfUserClient(userClient)
+            // when
```

Comment:
Optional: eventually we'll have a code formatter.
```suggestion
        // when
```